xml/admin_ceph_gateway.xml

```diff
@@ -3047,115 +2977,89 @@ us-east-1.rgw.meta</screen>
-     Update the period and commit the changes:
-    </para>
-<screen>&prompt.cephuser;radosgw-admin period update --commit
-{
-  "id": "b5e4d3ec-2a62-4746-b479-4b2bc14b27d1",
-  "epoch": 1,
-  "predecessor_uuid": "09559832-67a4-4101-8b3f-10dfcd6b2707",
-  "sync_status": [ "[...]"
-  ],
-  "period_map": {
-      "id": "b5e4d3ec-2a62-4746-b479-4b2bc14b27d1",
-      "zonegroups": [
-          {
-              "id": "d4018b8d-8c0d-4072-8919-608726fa369e",
-              "name": "us",
-              "api_name": "us",
-              "is_master": "true",
-              "endpoints": [
-                  "http:\/\/rgw1:80"
-              ],
-              "hostnames": [],
-              "hostnames_s3website": [],
-              "master_zone": "83859a9a-9901-4f00-aa6d-285c777e10f0",
-              "zones": [
-                  {
-                      "id": "83859a9a-9901-4f00-aa6d-285c777e10f0",
-                      "name": "us-east-1",
-                      "endpoints": [
-                          "http:\/\/rgw1:80"
-                      ],
-                      "log_meta": "true",
-                      "log_data": "false",
-                      "bucket_index_max_shards": 0,
-                      "read_only": "false"
-                  }
-              ],
-              "placement_targets": [
-                  {
-                      "name": "default-placement",
-                      "tags": []
-                  }
-              ],
-              "default_placement": "default-placement",
-              "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7"
-          }
-      ],
-      "short_zone_ids": [
-          {
-              "key": "83859a9a-9901-4f00-aa6d-285c777e10f0",
-              "val": 630926044
-          }
-      ]
-  },
-  "master_zonegroup": "d4018b8d-8c0d-4072-8919-608726fa369e",
-  "master_zone": "83859a9a-9901-4f00-aa6d-285c777e10f0",
-  "period_config": {
-      "bucket_quota": {
-          "enabled": false,
-          "max_size_kb": -1,
-          "max_objects": -1
-      },
-      "user_quota": {
-          "enabled": false,
-          "max_size_kb": -1,
-          "max_objects": -1
-      }
-  },
-  "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7",
-  "realm_name": "gold",
-  "realm_epoch": 2
-}</screen>
-   </sect3>
+   <note>
+     <para>Updating the period changes the epoch and ensures that other
+       zones receive the updated configuration.</para>
+   </note>
+</sect3>
+   <sect3 xml:id="update-ceph-config-file">
+     <title>Update the &ceph; Configuration File</title>
+     <para>Update the &ceph; configuration file on master zone hosts by adding
+       the <literal>rgw_zone</literal> configuration option and the name
+       of the master zone to the instance entry.</para>
+<screen>[client.rgw.<replaceable>INSTANCE-NAME</replaceable>]
+...
+rgw_zone=<replaceable>ZONE-NAME</replaceable></screen>
+     <para>For example:</para>
+<screen>[client.rgw.rgw1]
+host = rgw1
+rgw frontends = "civetweb port=80"
```

Comment:
NIT: What version of SES id this doc for? when are we moving to beast over civerweb. Not that this has anything to do with the scope of this doc.

Reply:
agree let's put beast here

Reply:
I have no idea what this means, fellas :) could you both elaborate further so I know exactly what I am including here?

Reply:
these are the 2 http frontends we support http://docserv.nue.suse.com/documents/SES_6/ses-admin/single-html/#ogw-http-frontends

Reply:
Thanks. Change to `rgw frontends = "beast port=80"`